### ISSUE TITLE
Search in Thread

### ISSUE THREAD

**Issue by binaryplease** [pazz/alot#830]
Hello,

I couldn't find this (essential) feature. Is there a possibility to search the text of a thread when viewing it (in thread-mode)?

Thanks

**Comment by pazz** [comment]
Not implemented yet. The problem is not finding the text
but to change the displayed widgets to highlight etc.

**Comment by lucc** [comment]
Maybe for starters you could leave out the highlighting and just jump to the relevant line, much like #806.

**Comment by pazz** [comment]
no: this jumps to a message, not a line. Jumping etc is easy once you have separated the line you want from the rest. It is this mechanism that we lack currently.

**Comment by matclab** [comment]
A workaround is to use `:pipeto --al less` allows to use `/` and `?` to search the thread content.
